src/test/java/org/kohsuke/github/AppTest.java

```diff
@@ -234,16 +234,16 @@ private boolean shouldBelongToTeam(String organizationName, String teamName) thr
         return team.hasMember(gitHub.getMyself());
     }
 
-    @Ignore("Needs mocking check")
     @Test
     public void testShouldFetchTeam() throws Exception {
-        GHOrganization j = gitHub.getOrganization(GITHUB_API_TEST_ORG);
-        GHTeam teamByName = j.getTeams().get("Core Developers");
+        GHOrganization organization = gitHub.getOrganization(GITHUB_API_TEST_ORG);
+        GHTeam teamByName = organization.getTeams().get("Core Developers");
 
-        GHTeam teamById = gitHub.getTeam(teamByName.getId());
```

Comment:
Sorry, just because it is deprecated doesn't mean we can remove the test for it.  We must ensure it continues to work until it is removed.